src/main/java/seedu/mtracker/console/EditInstrumentParser.java

```diff
@@ -79,6 +111,13 @@ public void editRemarksParameter(HashSet<String> parametersGiven)
         editedParameters.put(REMARK_ATTRIBUTE, inputRemark);
     }
 
+    /**
+     * Gets a new past return if past return is being edited and stores it.
+     *
+     * @param parametersGiven All the parameters to be edited.
+     *
```

Comment:
Sry no need space between the @ stuff, I was referring to the space between the description text and the first @